### DISCUSSION TITLE
xDrip+ Documentation

### DISCUSSION THREAD

**Discussion by Lupum1001** [NightscoutFoundation/xDrip#2158] [General]
Would it be possible to link to https://xdrip.readthedocs.io/en/latest/ in the Wiki and the main README.md file?

I'm sure there's a much bigger discussion to be had about xDrip's documentation, but for now readthedocs appears to at least be consistently maintained, so it would be helpful to include it with the other links.

**Comment by tolot27** [comment]
Yes, good idea. Please can you submit a PR, or @psonnera can do this.

**Reply by psonnera** [reply]
I would like to understand if it does add value or have more positive/negative feedback on it before pushing it. But I don't see any problem in having it mentioned in the repo .md.

**Reply by Lupum1001** [reply]
I only really knew about it, because I've seen @Navid200 posting links to various topics in it. Is he the maintainer?

**Reply by psonnera** [reply]
Navid maintains https://navid200.github.io/xDrip/ that is generic and Dexcom oriented.

**Reply by Navid200** [reply]
I am happy to open a PR and add your and my documentation as links.

**Reply by psonnera** [reply]
Sounds good

**Reply by Navid200** [reply]
Actually, I cannot open a PR for the wiki.  I keep forgetting.  Only two individuals can edit the wiki, as far as I know.  @tolot27

**Reply by Navid200** [reply]
@tolot27 This is my suggestion for the main page of the wiki, leaving the sidebar as is for now:  
https://github.com/Navid200/xDrip/wiki/Suggestion-for-Main-Wiki

What do you think?

**Reply by Navid200** [reply]
I will open a PR for the README.md.
